doc/source/tutorials-website/tutorials.html.jinja

```diff
@@ -0,0 +1,40 @@
+<div class="row">
+<div class="col-md-12">
+<h1>{{ title }}</h1><br>
+</div>
+</div>
+
+<h4 id="quantum-information-processing">Quantum information processing</h4>
+<p>This section contains tutorials for QuTiP Version 5. You can find the tutorials for QuTiP Version 4 <a href="https://qutip.org/qutip-tutorials/">here</a>.
```

Comment:
Could you add here one more sentence:

You can check the qutip version via ``qutip.about()``.

Reply:
yes